v0/destinations/yahoo_dsp/util.js

```diff
@@ -0,0 +1,152 @@
+const qs = require("qs");
+const { generateJWTToken } = require("../../../util/jwtTokenGenerator");
+const { httpSend } = require("../../../adapters/network");
+const { CustomError } = require("../../util");
+
+const { ACCESS_TOKEN_CACHE_TTL } = require("./config.js");
+const Cache = require("../../util/cache");
+
+const accessTokenCache = new Cache(ACCESS_TOKEN_CACHE_TTL);
+
+const getUnixTimestamp = () => {
+  return Math.floor(Date.now() / 1000);
+};
+
+const poiLocationType = [
+  "includeChains",
+  "includeWoeids",
+  "includeCategories",
+  "includeGids",
+  "excludeChains",
+  "excludeWoeids",
+  "excludeCategories",
+  "excludeGids"
+];
+let listType;
+const includes = {};
+const excludes = {};
+
+const populateIncludes = (list, audienceType) => {
+  list.forEach(element => {
```

Comment:
`list` doesn't tell another developer what we are trying to do. I mean it doesn't explain what that `list` nor does it communicate what the `list` an array(probably) might contain as well. I'd recommend you choose names that expresses the nature of the object in question.

The idea is to make it sound like prose for some other developer who reads your code

Reply:
Updated it with a relevant name "audienceList"

Reply:
The same goes with naming of `keys` in the later parts of this file. Please make necessary changes as required